doc/SDK_cross_compile_example.md

```diff
@@ -63,7 +63,7 @@ You can use *export -p* to verify RPI\_ROOT has been added to the environment.
 
 Now we need to switch to the SDK directory tree. Enter this command
 ```
-cd ~/Source/azure-iot-sdks/c/build_all/linux
+cd ~Source/azure-iot-sdk-c/build_all/linux
```

Comment:
It should be ~/Source/...
Like ~/Source/azure-iot-sdk-c/build_all/linux

Reply:
Fixed